packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1699,13 +1694,40 @@ function commitRootImpl(root) {
       currentTime,
       remainingExpirationTime,
     );
+
+    let prevInteractions: Set<Interaction> | null = null;
+    if (enableSchedulerTracing) {
+      if (remainingExpirationTime === Never) {
+        // Temporarily restore interactions in the case of e.g. hidden subtrees and suspended hydration.
+        // Don't restore otherwise or it would blur interrupting high-pri udpates with low-pri work.
```

Comment:
I think this is the right approach to trace hidden work without blending high/low pri updates.

Reply:
Hmm... this might still cause undesirable blurring if the idle/never work was there _before_ a high pri interruption. That seems harder to prevent. Is it worth trying to?